### PR TITLE
Correct variants of "debug" with three g's

### PR DESCRIPTION
**Pull request by dkg**
https://codesearch.debian.net/search?q=debuggg shows 8 pages of
results for variants of "debug" with three g's.

I've tried to capture them all here.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8138,6 +8138,12 @@ debufs->debugfs
 debugee->debuggee
 debuger->debugger
 debugg->debug
+debuggg->debug
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:8141]: Please can you add `debugggs->debugs` too.

**Review by peternewman — CHANGES_REQUESTED** [review]
One comment

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (bl-ue) [reply codespell_lib/data/dictionary.txt:8141]: → 8b54cf7f6a6b4e1e3a2010e551604982e9c029b5

**Review by bl-ue — APPROVED** [review]

**Comment by bl-ue** [comment]
Applied changes, LGTM.  PTAL @peternewman.

**Pull request merged**